src/plone/restapi/tests/http-examples/expansion_expanded_full.resp

```diff
@@ -9,100 +9,118 @@ Content-Type: application/json
                 {
                     "icon": "",
                     "id": "view",
-                    "title": "View"
+                    "title": "View",
+                    "url": "http://localhost:55001/plone/front-page"
```

Comment:
please add a trailing `,` in order to avoid a two-line diff next time (same as `black` would do).

Reply:
just found, `black` check is failing anyway.

Reply:
added running black. That shoukd be catched by the black checker indeed :/